yt_dlp/extractor/twitter.py

```diff
@@ -471,10 +453,10 @@ class TwitterIE(TwitterBaseIE):
             'timestamp': 1442188653,
             'upload_date': '20150913',
             'uploader_url': 'https://twitter.com/freethenipple',
-            'comment_count': int,
-            'repost_count': int,
+            'comment_count': None,  # only available with auth
+            'repost_count': None,  # same
```

Comment:
```suggestion
            # comment_count, repost_count, view_count are only available with auth
```

All None check can be removed. This one comment is sufficient.